app/views/exports/hamilton_lrc_pbmc_bank_to_lrc_bank_seq_and_spare.csv.erb

```diff
@@ -32,7 +32,7 @@
     next if source_well.downstream_tubes.empty?
 
     # expecting 1 sample tube
-    destination_tube = source_well.downstream_tubes.first
+    destination_tube = source_well.downstream_tubes.last
```

Comment:
We talked about state checking after `source_well.downstream_tubes` and then calling `last` and 
skipping if we can't find the destination tube.

Reply:
I've realised there is already a check on the parent well state being 'passed' or we skip the position.
A non-passed well should not have created any chid tubes anyway.
For passed wells at this point the child tubes should have nothing in them and should be in 'pending' state, which is based on the state of the transfer requests into them (the transfer has not happened yet so pending).
So a state check on the tube.state = pending or skip should be all we need for now until we add the uuid stories